pkg/kv/kvserver/batcheval/cmd_refresh_range.go

```diff
@@ -15,25 +15,15 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/batcheval/result"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
-	"github.com/cockroachdb/cockroach/pkg/settings"
 	"github.com/cockroachdb/cockroach/pkg/storage"
 	"github.com/cockroachdb/cockroach/pkg/storage/enginepb"
-	"github.com/cockroachdb/cockroach/pkg/util"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
 	"github.com/cockroachdb/cockroach/pkg/util/uuid"
 	"github.com/cockroachdb/errors"
 )
 
-// refreshRangeTBIEnabled controls whether we use a TBI during ranged refreshes.
-var refreshRangeTBIEnabled = settings.RegisterBoolSetting(
-	settings.SystemOnly,
-	"kv.refresh_range.time_bound_iterators.enabled",
```

Comment:
Do we need to add this and other settings to the retiredSettings map in pkg/settings/registry.go?

Reply:
Ah, didn't know about that -- thanks for the heads-up!